staticd/static_nb.c

```diff
@@ -60,7 +60,9 @@ const struct frr_yang_module_info frr_staticd_info = {
 		{
 			.xpath = "/frr-routing:routing/control-plane-protocols/control-plane-protocol/frr-staticd:staticd/route-list/path-list/frr-nexthops/nexthop",
 			.cbs = {
+#ifndef INCLUDE_MGMTD_VALIDATE_ONLY
```

Comment:
Is there any reason for linking `static_nb.c` into mgmtd other than doing mgmtd-side validation?

Reply:
I think if we remove the need for local validations on MGMTd this file doesn't need to be linked with MGMTd anymore. Will try it out. But first we need consesnsus on wether to keep local validations and remove remote validations, or vice versa. Hoping we can hash this out first in the upcoming meeting I requested.

Reply:
It uses it to determine what the component is "subscribed" to I believe. I mentioned in slack that I think it would be passable to move the ifdefs to this file to remove the callback references (so that code doesn't need to be linked in) basically b/c this file is not pretty to start and doesn't really contain functional code (it's just a generated structure) so they are a lot less disruptive.

Reply:
FWIW the git patch I submitted to fix remote validation didn't remove local validation from mgmtd, it was very little code to leave it in there. My aim here isn't to remove local validation, but rather to not use it by default and have the initial review and porting effort use remote validation b/c it minimized the component porting changes and review burden (for this PR as well as future porting PRs)